### PR TITLE
Connect to dry run cluster

### PR DESCRIPTION
**Pull request by deepfire**

### CODE REVIEW

**Comment by deepfire** [comment]
cc @domenkozar

File: scripts/launch/connect-to-cluster/dry-run-1.0.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+stack exec -- cardano-node                                  \
+    --tlscert ./scripts/tls-files/server.crt                \
+    --tlskey ./scripts/tls-files/server.key                 \
+    --tlsca ./scripts/tls-files/ca.crt                      \
+    --no-ntp                                                \
+    --topology "${TMP_TOPOLOGY_YAML}"                       \
+    --log-config scripts/log-templates/log-config-qa.yaml   \
+    --logs-prefix "logs/${CLUSTER}"                         \
+    --db-path db-${CLUSTER}                                 \
+    --wallet-db-path wdb-${CLUSTER}                         \
```
Comment (domenkozar) [inline scripts/launch/connect-to-cluster/dry-run-1.0.sh]: typo `wdb`

**Comment by domenkozar** [comment]
Hydra fails: https://hydra.iohk.io/build/35815/nixlog/2/tail

File: scripts/launch/connect-to-cluster/dry-run-1.0.sh (hunk 1 above)
Comment (deepfire) [reply scripts/launch/connect-to-cluster/dry-run-1.0.sh]: @domenkozar, why do you think it is?

Comment (domenkozar) [reply scripts/launch/connect-to-cluster/dry-run-1.0.sh]: D'oh - ignore.

**Pull request closed without merging**